src/overlays/actors/ovl_En_Encount3/z_en_encount3.h

```diff
@@ -7,10 +7,36 @@ struct EnEncount3;
 
 typedef void (*EnEncount3ActionFunc)(struct EnEncount3*, PlayState*);
 
+#define ENCOUNT3_GET_SPAWN_INDEX(thisx) (((thisx)->params >> 0xC) & 0xF)
+#define ENCOUNT3_GET_PARAM_F80(thisx) (((thisx)->params >> 7) & 0x1F)
+#define ENCOUNT3_GET_SWITCHFLAGS(thisx) ((thisx)->params & 0x7F)
+
+#define ENCOUNT3_GET_ROTZ(thisx) ((thisx)->world.rot.z)
+
 typedef struct EnEncount3 {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnEncount3ActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x84];
+    /* 0x148 */ u8 unk148;
+    /* 0x14A */ s16 unk14A;
+    /* 0x14C */ UNK_TYPE1 unk14C[0x2];
```

Comment:
```suggestion
    /* 0x14C */ UNK_TYPE1 pad14C[0x2];
```
Seems unlikely its used by a second actor when our actor doesn't use it, could label as a pad.